dnscrypt-proxy/example-allowed-names.txt

```diff
@@ -1,16 +1,16 @@
 
 ###########################
-#        Whitelist        #
+#        Allowlist        #
 ###########################
 
-## Rules for name-based query whitelisting, one per line
+## Rules for name-based query allowlisting, one per line
 ##
 ## Example of valid patterns:
 ##
 ## ads.*         | matches anything with an "ads." prefix
 ## *.example.com | matches example.com and all names within that zone such as www.example.com
 ## example.com   | identical to the above
-## =example.com  | whitelists example.com but not *.example.com
+## =example.com  | allowlists example.com but not *.example.com
```

Comment:
allows

Reply:
used "allows example.com but not *.example.com"